tests/muxcable_test.py

```diff
@@ -416,6 +421,72 @@ def test_config_muxcable_tabular_port_with_incorrect_port(self):
 
         assert(result.exit_code == 1)
 
+    @mock.patch('os.geteuid', mock.MagicMock(return_value=0))
+    @mock.patch('sonic_y_cable.y_cable.get_eye_info', mock.MagicMock(return_value=[0, 0]))
+    def test_show_muxcable_eye_info(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(show.cli.commands["muxcable"].commands["eyeinfo"], [
+                               "0", "0"], obj=db)
+
+        assert(result.exit_code == 0)
+
+    @mock.patch('os.geteuid', mock.MagicMock(return_value=0))
+    @mock.patch('sonic_y_cable.y_cable.get_ber_info', mock.MagicMock(return_value=[0, 0]))
+    def test_show_muxcable_ber_info(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(show.cli.commands["muxcable"].commands["berinfo"], [
+                               "0", "0"], obj=db)
+
+        assert(result.exit_code == 0)
+
+    @mock.patch('os.geteuid', mock.MagicMock(return_value=0))
+    @mock.patch('sonic_y_cable.y_cable.enable_prbs_mode', mock.MagicMock(return_value=1))
+    def test_config_muxcable_enable_prbs(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(config.config.commands["muxcable"].commands["prbs"].commands["enable"], [
+                               "0", "0", "0", "0"], obj=db)
+
+        assert(result.exit_code == 100)
+
+    @mock.patch('os.geteuid', mock.MagicMock(return_value=0))
+    @mock.patch('sonic_y_cable.y_cable.enable_loopback_mode', mock.MagicMock(return_value=1))
+    def test_config_muxcable_enable_loopback(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(config.config.commands["muxcable"].commands["loopback"].commands["enable"], [
+                               "0", "0", "0"], obj=db)
+
+        assert(result.exit_code == 100)
+
+    @mock.patch('os.geteuid', mock.MagicMock(return_value=0))
+    @mock.patch('sonic_y_cable.y_cable.disable_prbs_mode', mock.MagicMock(return_value=1))
+    def test_config_muxcable_disble_prbs(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(config.config.commands["muxcable"].commands["prbs"].commands["disable"], [
+                               "0", "0"], obj=db)
+
+        assert(result.exit_code == 100)
+
+    @mock.patch('os.geteuid', mock.MagicMock(return_value=0))
+    @mock.patch('sonic_y_cable.y_cable.disable_loopback_mode', mock.MagicMock(return_value=1))
+    def test_config_muxcable_disable_loopback(self):
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(config.config.commands["muxcable"].commands["loopback"].commands["disable"], [
+                               "0", "0"], obj=db)
```

Comment:
Suggest breaking the line after the comma.

Reply:
fixed